### PR TITLE
error_handling: add error_handler and better logging

### PR DESCRIPTION
**Pull request by ada-phillips**
Add new ErrorHandler class
Route ErrorHandler methods as flutterError.onError and platformDispatcher .onError handler
Modify Logger to write to stdout as well as log file Change default log file location
Refactor api_services methods to log more information regarding api calls, and surface exceptions
Add missing api methods (manual home, manual command, config) Fix delete api method

### CODE REVIEW

File: lib/api_services/api_services.dart [hunk 1]
```diff
@@ -18,17 +18,22 @@
...
-  static const String apiUrl = kDebugMode ? "https://dev.plyktra.de" : 'http://127.0.0.1:12357';
+  static const String apiUrl =
+      kDebugMode ? "https://dev.plyktra.de" : 'http://127.0.0.1:12357';
 
   // Method for creating a Uri object based on http or https protocol
-  static Uri dynUri(String apiUrl, String path, Map<String, dynamic> queryParams) {
+  static Uri dynUri(
+      String apiUrl, String path, Map<String, dynamic> queryParams) {
     if (queryParams.containsKey('file_path')) {
-      queryParams['file_path'] = queryParams['file_path'].toString().replaceAll('//', '');
```
Comment (ada-phillips) [inline lib/api_services/api_services.dart:31]: These changes were just `dart format` changes.

Comment (PaulGD03) [reply lib/api_services/api_services.dart:31]: all good

**Comment by PaulGD03** [comment]
looks good, lgtm!

**Review by PaulGD03 — COMMENTED** [review]
👍🏻

**Pull request merged**